tests/integration/test_juju_info_cos_agent.py

```diff
@@ -12,6 +12,7 @@
 from pytest_operator.plugin import OpsTest
 
 agent = SimpleNamespace(name="agent")
+hwo = SimpleNamespace(charm="hardware-observer", name="hwo")
```

Comment:
`hwo` added but no new assertions. Is that intentional?

Reply:
We are checking the existence of missing `juju_unit` label here:

```python
    for targets in unit_targets:
        for target in targets:
            target_labels = target["labels"].keys()
            assert topology_labels.issubset(target_labels)
```

Do you think we can be more specific? 🤔